chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -494,23 +494,49 @@ mcCombinations.prototype.possibleCombinations = [
 		maxfields: 1,
 		score: 0,
 		autoSubmit: false
+	},
+	{
+		combinationId: 'form001',
```

Comment:
it looks to me that you're completely changing the way we detect forms with this new maxfields value and scorepermatch value... so I'm afraid this is not acceptable.
Such drastic changes to our detection algo have massive repercussions.